apps/condo/domains/news/components/RecipientCounter.tsx

```diff
@@ -28,12 +36,13 @@ interface CounterProps {
     value: number
     type?: TypographyTitleProps['type'],
     hint?: string
+    downloader?: any
```

Comment:
And it is not an any type for the react element. It would be great, if you provide here a correct type